### PR TITLE
Réduction de la taille du texte explicatif pour le contact

### PR DESCRIPTION
**Pull request by alemangui**
Après :
image

Avant : 
image

### CODE REVIEW

File: frontend/src/views/CanteensPage/ContactForm.vue [hunk 1]
```diff
@@ -1,55 +1,49 @@
...
-      <p class="mt-4">Il ne s'agit cependant pas de :</p>
-      <div class="ml-2">
-        <p class="mb-1">
-          <v-icon small color="red">$close-line</v-icon>
-          postuler à une offre d'emploi pour une cantine => privilégier des contacts de l'établissements hors de ma
-          cantine
+      <div class="mt-4">
+        <p class="mb-1 d-flex align-center">
+          <v-icon small color="amber darken-3" class="mr-1">$error-warning-line</v-icon>
+          Privilégiez les contacts directs de l'établissement pour postuler à une offre d'emploi
```
Comment (hfroot) [inline frontend/src/views/CanteensPage/ContactForm.vue:18]: I liked the original structure which put the user need up front, making it a bit easier to scan the list to identify where your need is.
Maybe these sentences can be rewritten like:
pour X, faire Y
e.g.
"pour postuler à une offre d'emploi, privilégiez..."
"pour informer de l'absence ou présence de soi ou d'un tiers, utilisez la vie scolaire"

**Review by hfroot — COMMENTED** [review]
I also made a commit to vertically centre the icons

File: frontend/src/views/CanteensPage/ContactForm.vue (hunk 1 above)
Comment (alemangui) [reply frontend/src/views/CanteensPage/ContactForm.vue]: Reformulation ici : https://github.com/betagouv/ma-cantine/pull/2257/commits/594c8a3cdf567acfc1244b5497ea89d9d923ee76

File: frontend/src/views/CanteensPage/ContactForm.vue [hunk 2]
```diff
@@ -1,55 +1,49 @@
...
+          Proposez vos services en vous référençant parmis les
+          <router-link :to="{ name: 'PartnersHome' }" class="mx-1">
             acteurs de l'éco-système
           </router-link>
-          en
-          <a href="https://startupdetat.typeform.com/to/JhhsMCYC" target="_blank" rel="noopener">
-            demandant votre référencement
+          en cliquant
+          <a href="https://startupdetat.typeform.com/to/JhhsMCYC" target="_blank" rel="noopener" class="mx-1">
+            ici
```
[the remark this answers is not in the document]
Comment (alemangui) [reply frontend/src/views/CanteensPage/ContactForm.vue]: Reforumlation ici : https://github.com/betagouv/ma-cantine/pull/2257/commits/594c8a3cdf567acfc1244b5497ea89d9d923ee76

**Review by hfroot — COMMENTED** [review]
https://github.com/betagouv/ma-cantine/commit/594c8a3cdf567acfc1244b5497ea89d9d923ee76#commitcomment-97904508

**Comment by alemangui** [comment]
@hfroot reformulation concernant https://github.com/betagouv/ma-cantine/pull/2257#pullrequestreview-1267737762 ici : https://github.com/betagouv/ma-cantine/pull/2257/commits/6fcd110f5cc5580fcdf40a728349f09b42ab3995

**Review by hfroot — APPROVED** [review]

**Pull request merged**